internal/api/dependency/service_manager.go

```diff
@@ -102,6 +105,7 @@ type service struct {
 	secret             secret.Service
 	user               user.Service
 	variable           variable.Service
+	view               view.Service
```

Comment:
from the issue, it mentioned an endpoint `visit` and here it's `view`. I don't really mind to change the name, just curious about the reason you changed it :).

Reply:
TBH, no particular reason. I completely forgot I named it visit in the issue :|